ReactCommon/cxxreact/SystraceSection.h

```diff
@@ -9,6 +9,12 @@
 #include <fbsystrace.h>
 #endif
 
+#ifdef __OBJC__
+#define RCT_UNUSED ___unused
```

Comment:
Good to improve portability 👍 , just only `__OBJC__` may defined this macro? I think it's the compiler attribute, `GCC` or other compiler not support?

Reply:
And it's `__unused`, not `____unused`.

Reply:
I messed that up, thanks for pointing that out! The fact that this didn't cause any CI failures means that `ifdef __OBJC__` wasn't even working on iOS, so I have to figure something else out.

As for other platforms, C++17 has the `[[maybe_unused]]` attribute and GCC has the `__attribute__((unused))` that could possibly be added in to this macro on available platforms

Reply:
`__unused` is simply a macro, so I tweaked it to just check for its definition instead.